src/defineFunction.js

```diff
@@ -223,6 +228,10 @@ export function defineFunctionBuilders<NODETYPE: NodeType>({
     });
 }
 
+export const normalizeArgument = function(arg: AnyParseNode): AnyParseNode {
+    return arg.type === "ordgroup" && arg.body.length === 1 ? arg.body[0] : arg;
+};
```

Comment:
Is it possible that `arg` could be a series of nested `ordgroup`s?  If so, would we want to unwrap each of them?

Reply:
@kevinbarabash I'm not sure whether it's possible, but that's not caused by changes in this PR.